client/packages/programs/src/JsonForms/components/DateOfBirth.tsx

```diff
@@ -94,7 +91,7 @@ const UIComponent = (props: ControlProps) => {
     setDoB(dob);
     handleChange(dobPath, formatDateTime.customDate(dob, 'yyyy-MM-dd'));
     handleChange(estimatedPath, true);
-    setCustomError(undefined);
+    customErrors.remove(path);
```

Comment:
If I change another field, the date value resets to previous valid value, save button enables, but it's still showing error: 

https://github.com/user-attachments/assets/743db670-c89a-4f58-a762-6d91954a7297